composer.json

```diff
@@ -18,8 +18,8 @@
   "require-dev": {
     "guzzlehttp/guzzle": "^7",
     "guzzlehttp/psr7": "^2",
-    "openeuropa/code-review": "^2.0.0-alpha6",
```

Comment:
I changed this to 2.0, as the alpha caused random issues with `composer update --prefer-lowest`. I hope this doesn't break any backwards compatibility,

Reply:
it's ok we will remove it anyway